src/state/actionCreators/index.js

```diff
@@ -1,56 +1,46 @@
-/*Remember separation of concerns.
+/*Consider breaking this down into separate actionCreator files if it becomes overly crowded!
```

Comment:
The amount of information passed is very helpful to future tickets and gives a better understanding of how to go about any future information.